### PR TITLE
[Admin][UI] Add menu search input to the sidebar

### PR DESCRIPTION
**Pull request by kulczy**
https://github.com/Sylius/Sylius/assets/15385420/ef80d644-8c27-4278-8af1-7ca391dd678e

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -103,6 +103,7 @@ sylius:
             product_not_active_in_channel: The product is not yet activated in this channel.
         product_variant: Product variant
         sales_summary: Sales summary
+        search_menu_item: 'Search menu item'
```
Comment (CoderMaggie) [inline src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml]:
```suggestion
        search_menu: 'Search menu'
```

**Review by CoderMaggie — COMMENTED** [review]
WDYT of a slightly different styling for this input to make it less visible? 🤔

**Review by bartek-sek — DISMISSED** [review]

File: src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml (hunk 1 above)
Comment (loic425) [reply src/Sylius/Bundle/AdminBundle/Resources/translations/messages.en.yml]: I prefer "Search menu item" IMHO.

**Comment by kulczy** [comment]
After changes

Screenshot 2024-07-05 at 10 50 14

**Review by CoderMaggie — APPROVED** [review]

**Review by bartek-sek — APPROVED** [review]

**Review by mpysiak — APPROVED** [review]

**Pull request merged**